src/components/KeyPromises/index.js

```diff
@@ -29,7 +29,7 @@ function KeyPromises({ actionLabel, items, title, titleProps, ...props }) {
       }
     }
   }, [stepperRef, setActiveStep]);
-  const steps = items.length;
+  const steps = items.length === 1 ? 0 : items.length;
```

Comment:
`steps` should always equal the number of items... otherwise the arrow buttons (next/previous) will still work resulting in runtime error since `activeStep` will be greater than `steps`

Reply:
We shouldn't mess with steps  @nerdlet